src/transpiling/mod.rs

```diff
@@ -109,6 +109,7 @@ impl Default for EmitOptions {
 
 impl EmitOptions {
   fn as_typescript_strip_config(&self) -> typescript::strip::Config {
+    #![allow(deprecated)]
```

Comment:
What is this for?

Reply:
`use_define_for_class_fields` is a deprecated property, but we don't use it anyway because we target ES2021 (btw, should we increase that to 2022?). I can add a comment on the property.